mne/inverse_sparse/mxne_inverse.py

```diff
@@ -598,13 +608,30 @@ def tf_mixed_norm(evoked, forward, noise_cov, alpha_space, alpha_time,
     all_ch_names = evoked.ch_names
     info = evoked.info
 
+    if alpha is not None and l1_ratio is not None:
+        old_parametrization = False
+
+        if not (0. <= alpha <= 100.):
+            raise ValueError('alpha must be in range [0, 100].'
+                             ' Got alpha = %f' % alpha)
```

Comment:
Can you test if alpha == 100. and if so raise an Exception saying that no active source will be found